agrest/src/main/java/io/agrest/meta/DefaultAgRelationship.java

```diff
@@ -50,4 +51,11 @@ public boolean isToMany() {
     public PropertyReader getPropertyReader() {
         return propertyReader;
     }
+
+    @Override
+    public Object translateExpressionToTarget(Object Expression) {
+        return null;
```

Comment:
Code smell. Meaningless method in the class. This method only works in Cayenne implementation... I think this needs to be redesigned. Maybe it should stay at the Cayenne layer.